modules/tests/test-integration/src/test/java/org/ballerinalang/test/context/ServerInstance.java

```diff
@@ -60,18 +76,18 @@ public ServerInstance(String serverDistributionPath, int serverHttpPort) {
      * @throws Exception
      */
     @Override
-    public void start() throws Exception {
-        Utils.checkPortAvailability(httpServerPort);
-        if (serverHome == null) {
-            serverHome = setUpServerHome(serverDistribution);
-            log.info("Server Home " + serverHome);
-            configServer();
-        }
+    public void startServices() throws BallerinaTestException {
```

Comment:
Yes, it would make more sense to name these methods as startServer, stopServer and restartServer instead of naming them under services